mmseg/core/optimizers/layer_decay_optimizer_constructor.py

```diff
@@ -146,3 +173,22 @@ def add_params(self, params, module):
                 }
             logger.info(f'Param groups = {json.dumps(to_display, indent=2)}')
         params.extend(parameter_groups.values())
+
+
+@OPTIMIZER_BUILDERS.register_module()
+class LayerDecayOptimizerConstructor(LearningRateDecayOptimizerConstructor):
+    # Different learning rates are set for different layers of backbone.
+    # Note: Currently, this optimizer constructor is built for BEiT.
```

Comment:
```suggestion
    # Note: Currently, this optimizer constructor is built for BEiT, and it will be deprecated. Please use ``LearningRateDecayOptimizerConstructor`` instead.
```